aw/AppDelegate.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  AppDelegate.swift
+//  aw
+//
+//  Created by Alex Kerney on 3/21/18.
+//  Copyright © 2018 Alex Kerney. All rights reserved.
```

Comment:
You can delete all the autogenerated comments, this info is saved in the git history.

Reply:
Removed autogenerated comments in 
0eb5d49